🐞 `make environment` assumes unzip is installed

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

`make environment` script assumes `unzip` is installed, which can cause the operation to fail

```
unzip -o sample-data-1-0.zip
make[1]: unzip: Command not found
make[1]: *** [Makefile:589: fetch-sample-data] Error 127
make[1]: Leaving directory '/mnt/c/Users/John/Documents/GitHub/electionguard-core2'
make: *** [Makefile:121: environment] Error 2
```

### Expected Behavior

Operation completes with no error code.

### Steps To Reproduce

1. In Ubuntu 20.04 LTS
2. Git clone this repo
3. Run `make environment`

### Environment

```markdown
- OS:Ubuntu 20.04 LTS
```


### Anything else?

_No response_

--- a/Makefile
+++ b/Makefile
@@ -121,6 +121,7 @@ ifeq ($(OPERATING_SYSTEM),Linux)
 	sudo apt install -y clang-tidy
 	sudo apt install -y ninja-build
 	sudo apt install -y valgrind
+	sudo apt install -y unzip
 endif
 ifeq ($(OPERATING_SYSTEM),Windows)
 	@echo 🏁 WINDOWS INSTALL

